test/integration/testsuite/minishift.go

```diff
@@ -350,3 +350,26 @@ func (m *Minishift) minishiftHomeDirectoryShouldntExist(dir string) error {
 
 	return fmt.Errorf("Directory %s exists", dir)
 }
+
+func (m *Minishift) setEnvironmentVariable(variable string, value string) error {
+	os.Setenv(variable, value)
+	if os.Getenv(variable) != value {
+		return fmt.Errorf("Error in setting environment variable %s=%s in the current shell", variable, value)
+	}
+	return nil
+}
+
+func (m *Minishift) getEnvironmentVariable(variable string, value string) error {
+	if os.Getenv(variable) != value {
+		return fmt.Errorf("Environment variable value Expected: %s, Actual: %s in the current shell", value, os.Getenv(variable))
+	}
+	return nil
+}
+
+func (m *Minishift) unSetEnvironmentVariable(variable string) error {
+	os.Unsetenv(variable)
```

Comment:
Ditto https://golang.org/pkg/os/#Unsetenv need to handle error and don't need below code block.